### PR TITLE
Added Slovak language translation

### PR DESCRIPTION
**Pull request by Luc3as**
Added Slovak language files

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Luc3as, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#651).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: src/translations/translationMetadata.json [hunk 1]
```diff
@@ -92,6 +92,9 @@
 	"sl": {
 		"nativeName": "Slovenščina"
 	},
+	"sk": {
```
Comment (c727) [inline src/translations/translationMetadata.json]: sk has to be before sl

**Comment by c727** [comment]
Please use this platform for translations and remove sk.json from this PR: https://lokalise.co/project/3420425759f6d6d241f598.13594006/?view=multi

**Comment by Luc3as** [comment]
Okay, how do I add new language to lokalise?

Dňa 19. 11. 2017 9:16 odpoludnia používateľ "c727" <notifications@github.com>
napísal:

Please use this platform for translations and remove sk.json from this PR:
https://lokalise.co/project/3420425759f6d6d241f598.13594006/?view=multi

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/home-assistant/home-assistant-polymer/pull/651#issuecomment-345545992>,
or mute the thread

**Comment by c727** [comment]
I added you to Slovak now

**Comment by Luc3as** [comment]
Done !

**Pull request merged**